content/cluster/_index.md

```diff
@@ -1,10 +1,71 @@
 ---
 title: "Cluster"
-date: 2019-06-18T16:42:20+02:00
-lastmod: 2019-06-18T16:42:20+02:00
-publishdate: 2019-06-18T16:42:20+02:00
+date: 2020-04-21T15:22:20+02:00
+lastmod: 2020-04-21T15:22:20+02:00
+publishdate: 2020-04-21T15:30:20+02:00
 draft: false
 weight: 13
 ---
 
-Nothing to see here (yet)
+# Move from control-plane cluster to a control-plane and compute nodes cluster
+
+## Create computes
+
+```		
+oc label nodes ocp4-worker-0.example.com node-role.kubernetes.io/infra="" 
```

Comment:
shouldn't this be node-role.kubernetes.io/worker="" ?

Reply:
The official doc indicate it as infra:
https://docs.openshift.com/container-platform/4.3/machine_management/creating-infrastructure-machinesets.html#moving-resources-to-infrastructure-machinesets
Maybe worker also works, I didn't try it.